apps/docs/app/ui/components/propTable/propTable.css

```diff
@@ -51,34 +39,52 @@
     gap: var(--hd-space-1);
 }
 
-[data-column]::before {
-    display: var(--hd-props-table-cell-label-display);
-    content: attr(data-column);
-    flex: 0 0 5rem;
+.hd-props-table__cell.hd-table__cell {
+    border-bottom: var(--hd-props-table-cell-border);
     font-family: var(--hd-default-font-family);
-    font-size: 0.875rem;
-    font-weight: 700;
+    padding: 0;
 }
 
-.hd-props-table__cell.hd-table__cell {
-    padding-block: var(--hd-props-table-cell-padding-block);
-    padding-inline: var(--hd-props-table-cell-padding-inline);
-    border-bottom: var(--hd-props-table-cell-border);
+.hd-props-table__col-default {
+    color: var(--hd-color-neutral-text-weak);
 }
 
-.hd-props-table__col-name {
-    white-space: nowrap;
+.hd-props-table [data-rehype-pretty-code-figure] {
+    margin: 0;
+}
+
+.hd-props-table__description-list {
+    display: flex;
+    flex-direction: column;
 }
 
-.hd-props-table__col-description {
-    word-break: break-word;
+.hd-props-table__description-list .hd-props-table__description > *,
+.hd-props-table__default-value {
+    color: var(--hd-color-neutral-text-weak);
+    font-size: 0.875rem;
 }
 
-.hd-props-table__col-description p + * {
-    margin-block: unset;
+.hd-props-table__description-term {
+    display: flex;
+    gap: var(--hd-space-1);
 }
 
-.hd-props-table__col-description * {
-    font-size: var(--hd-props-table-cell-description-font-size);
+.hd-props-table__name {
+    font-family: var(--hd-default-font-family);
+    font-size: 0.875rem;
+    font-weight: 600;
+    line-height: 1.5;
 }
 
+.hd-props-table__cell.hd-props-table__col-default {
+    font-size: 0.875rem;
```

Comment:
can we have a prop for these? There are many of the same font-sizes

Reply:
Yes totally, well a variable ;p

Reply:
oops, yes variable xD my head is elsewhere I guess haha